docs/CLI.md

```diff
@@ -0,0 +1,51 @@
+# `plm` CLI reference
+
+If any required argument is ommitted, you will be asked to input manually.
+
+## `plm instance`
+
+Manage instances.
+
+### `plm instance new`
+
+Create a new local instance.
+
+**Example:**
+
+```bash
+plm instance new --private --domain plu.me --name 'My Plume Instance' -l 'CC-BY'
+```
+
+**Arguments:**
+
+- `--domain`, `-d`: the domain name on which your instance will be available.
+- `--name`, `-n`: The name of your instance. It will be displayed on the homepage.
+- `--default-license`, `-l`: the license to use for articles written on this instance, if no other license is explicitely specified. Optional, defaults to CC-0.
```

Comment:
this is not correct anymore

Reply:
Yes, but I will change everything in the same PR, so for now I used CC-0.